WIP -cold_outreach.md

```diff
@@ -0,0 +1,209 @@
+
+#Cold Outreach
+Reaching out by email to someone you don't know and does not know you can sometimes be disheartening. However, there are ways to increase the chances of getting a response and that is by finding the right person to contact and by using effective emails.
```

Comment:
@kazabooboo  I added a temporary intro. There needs to be something here for the reader to know what they will be getting in this chapter. Feel free to write what you would like here